client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -1507,7 +1508,7 @@ public void testGetDataFrameAnalyticsStats() throws Exception {
             .setAnalysis(OutlierDetection.createDefault())
             .build();
 
-        execute(
+        PutDataFrameAnalyticsResponse putConfigResponse = execute(
```

Comment:
Is this variable ever used?

Reply:
Well spotted. I ended up not using it and forgot to clean it up.